eslint.config.js

```diff
@@ -3,33 +3,27 @@
 import eslint from "@eslint/js";
 import jest from "eslint-plugin-jest";
 import prettier from "eslint-plugin-prettier";
+import svelte from "eslint-plugin-svelte";
+import svelteParser from "svelte-eslint-parser";
 import globals from "globals";
 import tseslint from "typescript-eslint";
 
-export default [
-    { files: ["**/*.ts"] },
+export default tseslint.config(
     { ignores: ["**/dist/"] },
     eslint.configs.recommended,
     ...tseslint.configs.recommended,
+    ...svelte.configs["flat/prettier"],
+    {
+        plugins: { jest, prettier, svelte },
+    },
     {
-        plugins: { jest, prettier },
-        languageOptions: {
-            globals: {
-                ...globals.browser,
-                ...jest.environments.globals.globals,
-            },
-            ecmaVersion: 2023,
-            sourceType: "module",
-            parserOptions: { project: "./tsconfig.json" },
-        },
         rules: {
             eqeqeq: "error",
             "prettier/prettier": "error",
             "no-console": "off",
             "no-plusplus": ["error", { allowForLoopAfterthoughts: true }],
             "no-unused-expressions": ["error", { allowShortCircuit: true }],
             "spaced-comment": ["error", "always", { markers: ["/"] }],
-            "@typescript-eslint/await-thenable": "error",
```

Comment:
?

Reply:
That rule was throwing an eslint error in the global rules scope at one point but it doesn't anymore so I'll move it back.